### ISSUE TITLE
all include files should be in a sirf subdir

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/SIRF#202]
to avoid conflicts, we should have to do
```c++
#include "sirf/...."
```

**Comment by evgueni-ovtchinnikov** [comment]
do you mean

    #include "sirf/pet/..."
    #include "sirf/mr/..."
    #include "sirf/common/..."

or everything in `sirf` without subfolders?

**Comment by KrisThielemans** [comment]
needs to be v2.0 as otherwise we're breaking backwards compatibility again

**Comment by KrisThielemans** [comment]
file structure
```
src/common/include/sirf/common/bla.h
src/cSTIR/include/sirf/STIR/bla.h
```

**Comment by KrisThielemans** [comment]
#193 uses `SIRF`. Probably best to stick to that.

**Comment by KrisThielemans** [comment]
actually, the namespace is `sirf`. In my opinion, this implies that the include dir should also be `sirf`.

**Comment by KrisThielemans** [comment]
Decided on `sirf`.

There's still a few include files that are imported without `sirf/` path (e.g. `iUtilities/data_handle.h`)

**Comment by evgueni-ovtchinnikov** [comment]
fixed by 0bd5123952e070aa52f43a458dcfd8705f23e5b2
